Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+numpy = "*"
+opencv-python = "*"
+streamlit = "*"
+pandas = "*"
+altair = "*"
+
+[requires]
+python_version = "3.6"
```

Comment:
Python 2.7 is also supported.

Reply:
I think it's better to use python 3*

https://github.com/pypa/pipfile/issues/87
https://pythonclock.org/
https://www.python.org/doc/sunset-python-2/

Reply:
Please, remove this constraint. It is a requirement for us to support Python 2.7 as well.

Reply:
Removed!